x-pack/legacy/plugins/apm/common/apm_saved_object_constants.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const APM_SERVICES_TELEMETRY_SAVED_OBJECT_TYPE =
+  'apm-services-telemetry';
+export const APM_SERVICES_TELEMETRY_DOC_ID = 'apm-services-telemetry';
+export const APM_INDICES_SAVED_OBJECT_TYPE = 'apm-indices';
+export const APM_INDICES_DOC_ID = 'apm-indices';
```

Comment:
WDYT about grouping them, and making the "id" constant naming consistent with the "type"?

```js
// APM Services telemetry
export const APM_SERVICES_TELEMETRY_SAVED_OBJECT_TYPE = 'apm-services-telemetry';
export const APM_SERVICES_TELEMETRY_SAVED_OBJECT_ID = 'apm-services-telemetry';

// APM indices
export const APM_INDICES_SAVED_OBJECT_TYPE = 'apm-indices';
export const APM_INDICES_SAVED_OBJECT_ID = 'apm-indices';
```